src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -450,14 +463,29 @@ public String getQuestionResultStatisticsCsv(
 
         DecimalFormat df = new DecimalFormat("#.##");
         
+        Map<String, List<Integer>> sortedOptionPoints = new TreeMap<String, List<Integer>>();
+        
+        Map<String, String> identifierMap = new HashMap<String, String>();
+        
         for (Entry<String, List<Integer>> entry : optionPoints.entrySet()) {
+            
+            if (distributeToRecipients) {
+                sortForRecipient(entry, identifierMap, sortedOptionPoints, bundle);
+            } else {
+                sortForOption(entry, options, sortedOptionPoints);
```

Comment:
Similarly, pass `optionPoints` into the method and bring the `for` loop into the method